terraform/groups/ecs-service/profiles/staging-eu-west-2/staging/vars

```diff
@@ -0,0 +1,15 @@
+environment = "staging"
+aws_profile = "staging-eu-west-2"
+
+# service configs
+use_set_environment_files = true
+log_level = "trace"
+
+# Scheduled scaling of tasks
+service_autoscale_enabled  = true
+service_scaledown_schedule = "55 19 * * ? *"
+service_scaleup_schedule   = "5 6 * * ? *"
+
+required_cpus = 512
+required_memory = 1024
```

Comment:
required_cpus & required_memory can be referred directly from the variables.tf file with the value of 512 & 1024